molecule/driver/openstackdriver.py

```diff
@@ -26,7 +26,10 @@
 import time
 
 import paramiko
-import shade
+try:
+    import shade
+except ImportError:
+    LOG.error('Driver missing, install shade!')
```

Comment:
LOG hasn't been imported at this point so this fails. In hindsight I think what we are really going to want is something like this.

```
try:
    import shade
except ImportError:
    sys.exit('ERROR: Driver missing, install shade!')
```

This way we don't require the LOG import and the program exits with a non-zero status.